pkg/controller/controller.go

```diff
@@ -112,11 +108,18 @@ func NewResizeController(
 		DeleteFunc: ctrl.deletePVC,
 	}, resyncPeriod)
 
-	podInformer.Informer().AddEventHandlerWithResyncPeriod(cache.ResourceEventHandlerFuncs{
-		AddFunc:    ctrl.addPod,
-		DeleteFunc: ctrl.deletePod,
-		UpdateFunc: ctrl.updatePod,
-	}, resyncPeriod)
+	if isVolumeInUserErrorHandlerEnabled() {
```

Comment:
I don't see any reason to not just `utilfeature.DefaultFeatureGate.Enabled(features.VolumeInUseErrorHandler)` directly here

Reply:
Done